### ISSUE TITLE
Restcomm-Connect does not support LetsEncrypt-enabled endpoints for callbacks

### ISSUE THREAD

**Issue by scottbarstow** [RestComm/Restcomm-Connect#2360]
If an external developer uses LetsEncrypt certificate for the SMS or Voice callback, Restcomm-Connect does not deliver the request. I am guessing that it's not included in the server's valid root CAs. 

I can provide a test url upon request if necessary.

**Comment by scottbarstow** [comment]
Also does not work for GeoTrust or Comodo. This needs to be fixed as it appears the list of trusted CAs (if this is in fact the issue) is getting smaller

**Comment by deruelle** [comment]
@scottbarstow given we use JDK 7, I think those root CA need to be added to the JDK trustcerts itself, this can potentially be done in the Docker image. @leftyb can you help @scottbarstow on this ?

**Comment by leftyb** [comment]
Yes definitely that is the problem, @gvagenas  helped to confirm on staging, by adding extra logs.
